Stylesheets/Docs/alert/docs_alert.html

```diff
@@ -7,61 +7,76 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Alerts | MondayBlues</title>
     <link rel="stylesheet" href="/Stylesheets/Abstract/fontawesome-free-6.0.0-web/css/all.css">
+    <link rel="shortcut icon" href="Assets/images/logo-modified.png" type="image/x-icon">
     <link rel="stylesheet" href="/Stylesheets/Docs/docs.css">
 </head>
 
 <body>
     <div class="page-contents">
         <nav class="display-align-center display-justify-space-between">
-            <div class="heading display-align-center display-justify-space-between">
+            <div class="heading display-align-center">
                 <div class="hamburger-nav clickable-image">
                     <div class="bars"></div>
                     <div class="bars"></div>
                     <div class="bars"></div>
                 </div>
-                <h1 class="title">mondayblues</h1>
+                <div class="image-container">
+                    <img src="/Assets/images/logo.png" alt="logo" class="logo-nav">
+                </div>
+                <h2 class="title">mondayblues</h2>
             </div>
             <div class="navbar">
-                <button class="nav-btn solid-secondary"><a href="https://github.com/anushasunil">gitHub</a> </button>
-                <button class="nav-btn solid-primary">Docs</button>
+                <button class="nav-btn solid-secondary"><a href="https://github.com/anushasunil/mondayblues">gitHub</a> </button>
+                <button class="nav-btn solid-primary"><a href="/index.html">Home</a></button>
             </div>
```

Comment:
This could have been a UI with Li inside it. Just a suggestion.

Reply:
Oh okay, I'll rectify it